metrics/report/README.md

```diff
@@ -0,0 +1,12 @@
+# PnP failure Report tool
+
+This tool reports by email the obtained PnP results from checktool
```

Comment:
In which case, this doc should probably refer to it as "`checkmetrics`, the check tool" or similar (and this PR should block on #98).